students/hoontae/users/models.py

```diff
@@ -0,0 +1,13 @@
+from django.db import models
+
+# Create your models here.
+class User(models.Model):
+    name      = models.CharField(max_length=45)
+    email     = models.CharField(max_length=300)
+    password  = models.CharField(max_length=400)
+    phone_num = models.CharField(max_length=20)
```

Comment:
```suggestion
    phone_number = models.CharField(max_length=45)
```
훈태님 변수명, Field 지정, 컨밴션 다 너무 좋습니다!! 하지만 변수명은 명확성을 위해 줄여쓰지 않으니 하나만 수정해주세요!